src/ontodia/diagram/view.ts

```diff
@@ -122,6 +124,34 @@ export class DiagramView extends Backbone.Model {
             this.model.resetHistory();
             this.zoomToFit();
         });
+        this.listenTo(this.selection, 'add remove reset', () => {
```

Comment:
Please, define an option in `DiagramViewOptions` to disable this Halo (to be able to use a `rappid` one).